jerry-core/vm/vm.c

```diff
@@ -65,30 +66,41 @@ static ecma_compiled_code_t *__program = NULL;
  */
 static ecma_value_t
 vm_op_get_value (ecma_value_t object, /**< base object */
-                 ecma_value_t property, /**< property name */
-                 bool is_strict) /**< strict mode */
+                 ecma_value_t property) /**< property name */
 {
   if (unlikely (ecma_is_value_undefined (object) || ecma_is_value_null (object)))
   {
     return ecma_raise_type_error (ECMA_ERR_MSG (""));
   }
 
-  ecma_value_t completion_value = ecma_make_simple_value (ECMA_SIMPLE_VALUE_EMPTY);
+  ecma_value_t completion_value = ecma_op_to_string (property);
```

Comment:
IMHO `completion_value` is a bad variable name here. It does not say anything about what contains. I suggest `prop_name_comp_value` instead or something like that.